### PR TITLE
Czech tiny fixies

### PR DESCRIPTION
**Pull request by ferben**
Only tiny fixies :)

### CODE REVIEW

File: Translations/cs.lproj/preferences.strings [hunk 1]
```diff
@@ -330,7 +330,7 @@
 "u2R-2w-aOA.title" = "Pro novou komprimaci, použít výchozí heslo";
 
 /* Class = "NSTextFieldCell"; title = "You can also use this password from the Finder Extension's menu"; ObjectID = "vWb-gb-JBD"; */
-"vWb-gb-JBD.title" = "Toto heslo lze také použít v kontextové nabídce Finderu";
+"vWb-gb-JBD.title" = "Toto heslo lze také použít v kontextové nabídce Finderu.";
```
Comment (aonez) [inline Translations/cs.lproj/preferences.strings:333]: That's a good one

**Comment by aonez** [comment]
Thanks again @ferben 🙌🏼

File: Translations/cs.lproj/preferences.strings (hunk 1 above)
Comment (ferben) [reply Translations/cs.lproj/preferences.strings:333]: This type of notes are consider as sentence, so it should end with punctuation (period, question or exclamation mark), so there should be period "." at the end also in English and all other languages.
In other places this rule is kept. See for example "File Access" or "Performance" tab in "Preferences".
ScreenShot 2023-09-05 at 12 02 53

Comment (ferben) [reply Translations/cs.lproj/preferences.strings:333]: Oh it's already done! Thank you

**Pull request merged**